sdcm/utils/common.py

```diff
@@ -3152,3 +3152,9 @@ def SoftTimeoutContext(timeout: int, operation: str):  # pylint: disable=invalid
     if duration > timeout:
         SoftTimeoutEvent(operation=operation, soft_timeout=timeout,
                          duration=duration).publish_or_dump()
+
+
+@contextmanager
+def DummyContextManager():  # pylint: disable=invalid-name
```

Comment:
I think it is redundant.
The `contextlib.nullcontext` does exactly the same and this lib is already used in the `sdcm/utils/nemesis_utils/indexes.py` module.

Reply:
thanks, I wasn't aware of it Fixed.